libcomposefs/lcfs-mount.c

```diff
@@ -666,6 +668,35 @@ static errint_t lcfs_mount(struct lcfs_mount_state_s *state)
 	return -EINVAL;
 }
 
+static int lcfs_mount_copy_and_seal(int fd) {
+	int memfd = memfd_create(CFS_MOUNT_SOURCE, MFD_CLOEXEC | MFD_ALLOW_SEALING);
```

Comment:
I'd use `cleanup_fd int memfd = -1;:` here and then `return steal_fd(&memfd)`; (Although i see we need to add a steal_fd for this.